OpenEdXMobile/src/main/java/org/edx/mobile/view/AccountFragment.kt

```diff
@@ -134,7 +134,7 @@ class AccountFragment : BaseFragment() {
                         })
                 videoQualityDialog.show(childFragmentManager, VideoDownloadQualityDialogFragment.TAG)
 
-                environment.analyticsRegistry?.trackVideoDownloadQualityClicked(
+                environment.analyticsRegistry?.trackEvent(
```

Comment:
Can we update the `environment.analyticsRegistry?.trackEvent` with local `trackEvent` method?

Reply:
Updated.